Release initial Auto SDK docs (#643)
autonomys/subspace-docs: docs/develop/intro.md

---
title: Getting Started for Developers
sidebar_position: 1
description: A Getting Started guide for the Autonomys Network
keywords:
    - Developers Documentation
    - DevDocs
    - SDK
    - Auto SDK
---

### Introduction
Autonomys is a secure, scalable, and decentralized blockchain that resolves the blockchain trilemma without making compromises. It offers a platform where developers can build applications with ease, leveraging innovative protocols and familiar tools. This guide provides an overview of the development options available on the Autonomys Network.

### Is It Difficult to Build Applications on Autonomys?

Absolutely not! Our primary objective is to minimize barriers to entry for both farmers and developers.

> We pride ourselves on offering unlimited possibilities—there are no boundaries!

Reviewer: ```suggestion
We pride ourselves on enabling unlimited possibilities - there are no boundaries!
```

## Development Options

**Developers** can choose to utilize **Auto SDK** - a comprehensive toolkit designed to simplify development on the Autonomys Network, or **Nova** - Autonomys's EVM-compatible domain, allowing you to utilize all the familiar functionalities available on networks like Ethereum. Soon, you'll also be able to build your own local custom virtual machines.

## [Auto SDK](/docs/develop/auto_sdk/intro)

### Key Features:

> - **Modular Packages**: The SDK is organized into packages like auto-utils, auto-consensus, and auto-id, allowing you to use only what you need.

Reviewer: ```suggestion
- **Modular Packages**: The SDK is organized into packages like auto-utils, auto-consensus, and auto-id. This partitions functionality intuitively and allows you to import only what you need.
```

- **Simplified Interactions**: Abstracts the complexity of blockchain operations with high-level functions.
- **Flexibility**: Suitable for building decentralized applications (dApps) and services with ease.

### Getting Started with Auto SDK
To start using Auto SDK, refer to our [Setup Instructions](/docs/develop/auto_sdk/intro.md) to clone the repository, install dependencies, and run tests. The SDK is designed to be developer-friendly, ensuring you can get up and running quickly.

## [Nova - EVM Compatible Domain](/docs/develop/nova/quick_start)

### Key Features:
- **Solidity Smart Contracts**: Write and deploy smart contracts using Solidity, just as you would on Ethereum.

> - **Tool Compatibility**: Any tool available for Ethereum development—like Remix, Truffle, or Hardhat—is compatible with Autonomys.

Reviewer: ```suggestion
- **Tool Compatibility**: Any tool available for Ethereum development - like Remix, Truffle, or Hardhat - is compatible with Nova.
```
